pkg/skyenv/dmsgpty.go

```diff
@@ -0,0 +1 @@
+package skyenv
```

Comment:
If I'm not mistaken we were going to call this `utilenv` correct?

Reply:
utilenv is the local reference to the skyenv import from skywire-utilities.

I'm not sure the purpose of this particular file, but the dmsgpty_unix.go file works with a path on the filesystem, and I think we should keep anything having to do with that in skywire.